src/core/ruby_version.rs

```diff
@@ -2925,4 +3288,64 @@ mod tests {
         .unwrap();
         assert_eq!(real_stack_trace_3_3_0(), stack_trace.unwrap().trace);
     }
+
+    #[rstest]
```

Comment:
I added rstest here for table tests on these label calculation functions.

The logic of the functions themselves **should** match their counterparts in cruby. 

In case we find cases where they don't, we can fix it and add regression tests here based on the input combos.

There is no need to update the test, just add new cases to the macro with the inputs to use and expected output.